doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -351,6 +351,13 @@ Matter samples
 * Updated all Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
```

Comment:
just noticed that when copying and pasting I've left this point here. it should be removed as it is covered under `Updated: `